MarathonRecomp/api/Marathon.h

```diff
@@ -136,6 +136,7 @@
 #include "Sonicteam/StdImageFilters/BurnoutBlurFilter.h"
 #include "Sonicteam/StdImageFilters/SingleTechniqueFilter.h"
 #include "Sonicteam/System/CreateStatic.h"
+#include "Sonicteam/System/Diagnostic/Perfomance.h"
```

Comment:
Make sure to change this file location if renaming from `Diagnostic` to `Diagnostics`.